<launch>
    <arg name="node_name" default="pcd_clipper"/>
    <arg name="enableLocalize" default="true"/>
    <arg name="input_topic" default="/points_raw_1"/>
    <arg name="input_frame_id" default="/velodyne_left"/>
    <!--<arg name="output_frame_id" default="/velodyne"/>-->
    <arg name="clipped_center_angle" default="0"/>
    <arg name="clipped_viewing_angle" default="360"/>
    <arg name="clipped_point_output_topic" default="/clipped_points_raw"/>
    <arg name="localization_point_output_topic" default="/localization_points_raw"/>    
    <arg name="localization_center_angle" default="0"/>
    <arg name="localization_viewing_angle" default="360"/>
    


    <node pkg="rubis_pkg" type="pcd_clipper" name="$(arg node_name)" output="screen">
        <param name="/enableLocalize" value="$(arg enableLocalize)" />
        <param name="/input_topic" value="$(arg input_topic)" />
        <param name="/input_frame_id" value="$(arg input_frame_id)" />
        <!--<param name="/output_frame_id" value="$(arg output_frame_id)" />-->
        <param name="/clipped_center_angle" value="$(arg clipped_center_angle)" />
        <param name="/clipped_viewing_angle" value="$(arg clipped_viewing_angle)" />
        <param name="/clipped_point_output_topic" value="$(arg clipped_point_output_topic)" />
        <param name="/localization_point_output_topic" value="$(arg localization_point_output_topic)" />        
        <param name="/localization_center_angle" value="$(arg localization_center_angle)" />
        <param name="/localization_viewing_angle" value="$(arg localization_viewing_angle)" />
    </node>
</launch>